acceptance-tests/lib/env.ts

```diff
@@ -66,7 +51,7 @@ const getTestConfig = () => {
   }
 
   if (!config.cliPath && !config.cliVersion) {
-    const defaultPath = path.join(process.cwd(), DEFAULT_CLI_PATH);
```

Comment:
`DEFAULT_CLI_PATH` was only used in one place and it's relative to the `cwd`, so it seem to make more sense to just inline it.